packages/flutterfire_ui/lib/src/i10n/default_localizations.dart

```diff
@@ -120,6 +121,7 @@ const localizations = <String, FlutterFireUILocalizationLabels>{
   'nl': NlLocalizations(),
   'id': IdLocalizations(),
   'de': DeLocalizations(),
+  'zh-CN': ZhCnLocalizations(),
```

Comment:
Do you mind changing this to just `zh`? It keeps it consistent with the rest of the language codes. This also means changing the class name to `ZhLocalizations`.

Reply:
Updated. You might encounter this issue again if someone contributes traditional Chinese in the future. :)